package.json

```diff
@@ -17,7 +17,7 @@
     "server:watch": "nodemon websocket_server/main.js"
   },
   "dependencies": {
-    "@excalidraw/excalidraw": "^0.17.6",
+    "@excalidraw/excalidraw": "^0.17.1-576bc0d",
```

Comment:
Is the bump of the release required for the title fix? If not I'd split this as i feel then the bump needs a bit more testing :)